packages/jest-resolve-dependencies/src/__tests__/dependency_resolver.test.ts

```diff
@@ -31,7 +35,7 @@ beforeEach(() => {
     },
     {},
   ).options;
-  return Runtime.createContext(config, {maxWorkers}).then(hasteMap => {
+  return Runtime.createContext(config, {maxWorkers}).then((hasteMap: any) => {
```

Comment:
This problem with haste map is really annoying me.
We could (in another PR) put `interface`s for the haste map classes into `@jest/types` and then reference those everywhere. However, once we migrate haste map to properly use a default export, they would become superfluous. @SimenB WDYT, should I try it that way?

Reply:
Yeah, let's do it